### PR TITLE
Clarify when we switch to running commands inside a master host.

### PR DESCRIPTION
**Pull request by cben**
Please verify this makes sense, I'm following these instructions for the first time in my life.

### CODE REVIEW

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -931,8 +931,8 @@ link:#verifying-the-installation[Verifying the Installation].
 == Verifying the Installation
 
 After the installer completes, you can verify that the master is started and
-nodes are registered and reporting in *Ready* status by running the following as
-*root*:
+nodes are registered and reporting in *Ready* status by sshing into the master
+host and running the following as *root*:
```
Comment (adellape) [inline install_config/install/advanced_install.adoc]: How about:

"After the installer completes, verify that the master is started and nodes are registered and reporting in Ready status. On the master host, run the following as root:"

Comment (cben) [inline install_config/install/advanced_install.adoc]: Amended (but decided to add emphasis *On the master host*).

It also crossed my mind that switching to full `root@master.example.com#` prompts could help, as they convey exactly the desired information - which user on which host.  But that might be more confusing than useful unless it's applied consistently across docs...

**Comment by adellape** [comment]
:+1: Thanks.

**Comment by adellape** [comment]
**revhistory note:**

(Install & Config -> Installing -> Advanced Installation)

```
Clarify in the link:../install_config/install/advanced_install.html#verifying-the-installation[Verifying the Installation] section to run the `oc get nodes` command on the master host.
```

**Pull request merged**